C/services/south/ingest.cpp

```diff
@@ -1206,11 +1206,27 @@ void Ingest::unDeprecateStorageAssetTrackingRecord(StorageAssetTrackingTuple* cu
 			datapoints,
 			count);
 
-	std::string data = "{\"datapoints\":[";
-	data.append(datapoints);
-        data.append("],\"count\":");
-        data.append(to_string(count));
-        data.append("}");
+	vector<string> tokens;
+        stringstream dpStringStream(datapoints);
+        string temp;
+        while(getline(dpStringStream, temp, ','))
+        {
+	        tokens.push_back(temp);
+        }
+
+	ostringstream convert;
+        convert << "{";
+        convert << "\"datapoints\":[";
+        for (unsigned int i = 0; i < tokens.size() ; ++i)
```

Comment:
There is probably no need to store tokens temporarily, you can tokenize and use tokens immediately.

Reply:
minor